plane/source/docs/guide-tailsitter.rst

```diff
@@ -151,40 +176,40 @@ and thrust vectoring in each flight mode.
 .. youtube:: s2KLOAdS_HY
     :width: 100%
 
-PID gain scheduling
-===================
+CopterMotor PID Gain Scaling
```

Comment:
a google search for "pid gain scheduling" produces more relevant results than one for "pid gain scaling"
But that's probably a dev-level concern :)

Reply:
scaling is used through-out the wiki and in hobbyist circles...made it consistent